pkg/framework/aries/framework.go

```diff
@@ -25,14 +25,16 @@ type DIDResolver interface {
 
 // Aries provides access to clients being managed by the framework.
 type Aries struct {
-	transport           api.TransportProviderFactory
-	didResolver         DIDResolver
-	storeProvider       storage.Provider
-	protocolSvcCreators []api.ProtocolSvcCreator
-	services            []dispatcher.Service
-	inboundTransport    transport.InboundTransport
-	walletCreator       api.WalletCreator
-	wallet              api.CloseableWallet
+	transport                 api.TransportProviderFactory
+	didResolver               DIDResolver
+	storeProvider             storage.Provider
+	protocolSvcCreators       []api.ProtocolSvcCreator
+	services                  []dispatcher.Service
+	inboundTransport          transport.InboundTransport
+	walletCreator             api.WalletCreator
+	wallet                    api.CloseableWallet
+	outboundDispatcherCreator dispatcher.OutboundCreator
+	outboundDispatcher        dispatcher.Outbound
```

Comment:
Second instance of this pattern inside this `Aries` struct: we're adding an attribute with a value provided by another one. In this case, this `outboundDispatcher` was created by `outboundDispatcherCreator`. Previously we added `services` being created by `procotolSvcCreators`. Are we *certain* that `Aries` needs to hold all these?

Reply:
I removed outboundDispatcher not needed aries to hold it